### PR TITLE
Track Ribbon treasury

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ribbon exports TVL: 
        
 
 
    ethereum-staking          7.90 M
    solana                    326.81 k
    avax                      864.33 k
    ethereum-treasury         18.50 M
    ethereum                  64.30 M
    staking                   7.90 M
    treasury                  18.50 M
    
    total                    65.49 M

File: projects/ribbon/evm.js [hunk 1]
```diff
@@ -106,6 +112,29 @@ async function avaxTvl(_, _b, { avax: block }) {
   return sumTokens2({ balances, owner: pauserAvax, tokens: [nullAddress, savax], chain, block, transformAddress, })
 }
 
+async function getTreasury(timestamp, block, chainBlocks) {
+  const balances = {};
+  await sumTokensAndLPsSharedOwners(
+    balances,
+    [
+        [RBN, false],
```
Comment (g1nt0ki) [inline projects/ribbon/evm.js:120]: Remove this line otherwise it looks good. We don't include project native tokens in treasury

**Comment by llamatester** [comment]
The adapter at projects/ribbon exports TVL: 
        
 
 
    solana                    316.29 k
    avax                      837.92 k
    ethereum-staking          7.36 M
    ethereum-treasury         2.17 M
    ethereum                  63.25 M
    staking                   7.36 M
    treasury                  2.17 M
    
    total                    64.40 M

**Comment by kargakis** [comment]
@g1nt0ki what's the best way to read the ETH balance of the treasury?

**Comment by llamatester** [comment]
The adapter at projects/ribbon exports TVL: 
        
 
 
    ethereum-staking          7.53 M
    solana                    305.96 k
    avax                      834.29 k
    ethereum                  59.20 M
    ethereum-treasury         2.48 M
    staking                   7.53 M
    treasury                  2.48 M
    
    total                    60.34 M

**Comment by g1nt0ki** [comment]
@kargakis thanks for the PR, I use `nullAddress`and pass it as part of token list. `sumTokens` method takes care of it (internally it calls `sdk.api.eth.getBalances` )

**Pull request merged**